burrow/datadog_checks/burrow/burrow.py

```diff
@@ -0,0 +1,228 @@
+from datadog_checks.base import AgentCheck
+from . import burrow_urls
+
+import requests
+
+CONFIG_BURROW_URL = "url"
+CONFIG_TIMEOUT = "timeout"
+CONFIG_CLUSTERS = "clusters"
+CONFIG_TAGS = "tags"
+
+TIMEOUT_DEFAULT = 20
+
+HEALTH_CHECK_NAME = "burrow.can_connect"
+
+
+class BurrowCheck(AgentCheck):
+    def check(self, instance):
+        burrow_url = instance.get(CONFIG_BURROW_URL, "http://localhost:8000")
+        burrow_clusters = instance.get(CONFIG_CLUSTERS, [])
+        extra_tags = instance.get(CONFIG_TAGS, [])
+        timeout = instance.get(CONFIG_TIMEOUT, TIMEOUT_DEFAULT)
+
+        self.log.debug("Starting check on burrow %s", burrow_url)
+        self._health_check(burrow_url, timeout, extra_tags)
+
+        clusters = self._find_clusters(burrow_url, burrow_clusters, timeout)
+
+        self.log.debug("Collecting Consumer Group Offsets")
+        self._consumer_groups_offsets(
+            burrow_url, clusters, timeout, extra_tags)
+
+        self.log.debug("Collecting Topic Offsets")
+        self._topic_offsets(burrow_url, clusters, timeout, extra_tags)
+
+        self.log.debug("Collecting Consumer Group lags")
+        self._consumer_groups_lags(
+            burrow_url, clusters, timeout, extra_tags)
+
+    def _health_check(self, burrow_url, timeout, extra_tags):
+        """
+        Check the Burrow health check endpoint
+        """
+        burrow_admin = burrow_urls.burrow_admin(burrow_url)
+        try:
+            tags = ["instance:%s" % self.hostname]
```

Comment:
hostname is already included by default as a `host` tag, no need to include it explicitly